runtimes/peregrine/src/benchmarks/mod.rs

```diff
@@ -0,0 +1,79 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2024 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use frame_benchmarking::define_benchmarks;
```

Comment:
The whole impl frame_benchmarking::Benchmark<Block> for Runtime
is missing for peregrine. Do we not still need that? 

Also the benchmark implementation for XCM is missing

Reply:
Ops! Big catch, thank you! I implemented the required functions in the benchmarking module and call into them from the runtime API impl module: https://github.com/KILTprotocol/kilt-node/pull/797/commits/e5c2f73f31dd17f2e6c44c287fba7e66c429cc12.

Reply:
Did you try running the benchmarks for the pallet? That would be a good indicator of whether we introduced any nasty bugs during the refactor.

Reply:
Ehm... I did now, and found out the issue can be resolved only by actually mocking the storage for the prime member. I did it here: https://github.com/KILTprotocol/kilt-node/pull/797/commits/f2c2d4629c53b4454bac23360a8dcdab38f0093e, and refactored it into its own module in the benchmarking module here: https://github.com/KILTprotocol/kilt-node/pull/797/commits/be50327e707d12a35e36cf16a7136789b68b9630

Reply:
And all other benchmarks did work? No strange spikes in the calculated weights?

Reply:
Did not run them on the benchmarking server. That is something we will do during the release. As far as I am concerned, all benchmarks worked, and no `verify` failed, so I'm good with that.